.github/workflows/release.yml

```diff
@@ -228,18 +275,30 @@ jobs:
           ziti_ci_gpg_key_id: ${{ secrets.ZITI_CI_GPG_KEY_ID }}
         shell: bash
         run: |
+
+          set -o xtrace
+
           $(go env GOPATH)/bin/ziti-ci configure-git
-          $(go env GOPATH)/bin/ziti-ci publish-to-github -t --prerelease --archive-base "" --dry-run
-          mkdir tmp
-          tar xfzv ./release/ziti-linux-amd64-* -C ./tmp
-          $(go env GOPATH)/bin/ziti-ci verify-current-version -t $(./tmp/ziti version)
-          $(go env GOPATH)/bin/ziti-ci publish-to-github -t --prerelease --archive-base ""
+
+          $(go env GOPATH)/bin/ziti-ci publish-to-github --use-current-tag --prerelease --archive-base "" --dry-run
+
+          tar xfzv ./release/ziti-linux-amd64-* -C /tmp
+          # TODO: should this compare current tag with "ziti --version" instead?
+          $(go env GOPATH)/bin/ziti-ci verify-current-version --use-current-tag "$(/tmp/ziti version)"
+
+          $(go env GOPATH)/bin/ziti-ci publish-to-github --use-current-tag --prerelease --archive-base ""
+
+      - name: Attest Build Provenance
+        uses: actions/attest-build-provenance@v2
+        with:
+          subject-checksums: ./release/attestation-subjects.sha256.txt
```

Comment:
This file is created by `ziti-ci publish-to-github` in the release artifact dir, but excluded from the release itself since it is only intended as input to GitHub's attestation Action.